test/TEST-30-ISCSI/99-default.link

```diff
@@ -0,0 +1,6 @@
+[Match]
```

Comment:
Same here as previously noted no longer the default and presumably this is the "client" link file so let's just call it client.link

Reply:
yes